source/tutorials/nix-shell-in-shebang.md

```diff
@@ -0,0 +1,33 @@
+# How to use nix-shel in a shebang
+
+The [nix
+shell](https://nixos.org/manual/nix/stable/command-ref/nix-shell.html)
+is feature provided by the [Nix package
+manager](https://nixos.org/guides/how-nix-works.html). It allows users
+to enter into a new shell environment containing a set of packages given
+on the command line.
```

Comment:
These lines are really hard to read in source. Please make use of [our markdown conventions](https://github.com/NixOS/nix.dev/blob/master/CONTRIBUTING.md?plain=1#L173-L185).